src/procedures/__tests__/ControllerTransfer.ts

```diff
@@ -191,7 +185,7 @@ describe('ControllerTransfer', () => {
         })
       )
     ).toEqual(true);
-    expect(await resolverValue).toEqual([undefined, undefined]);
+    await expect(resolverValue).toEqual([undefined, undefined]);
```

Comment:
Why await this?

Reply:
I await this because it gets rid of a compiler warning in the IDE. I thought I just needed to pull the await out

Reply:
"Promise returned from toEqual is ignored"